scripts/analyze.py

```diff
@@ -0,0 +1,28 @@
+import sys
+import os
+import pandas as pd
+
+from scripts.common import *
+from scripts.projects import Project
+from scripts.fuzzer_stats import FuzzerStats
+from scripts.util import summarize_fuzzer_stats_df
+
+
+def get_project_fuzzer_stats(project: str) -> pd.DataFrame:
+    fuzzout = "fuzz"
+    dumpout = "dump"
+    proj = Project(project, fuzzout, dumpout)
+    return proj.get_project_stats()
+
+
+def main():
+    projects = ["coturn", "json-c"]  # todo: full list of built & fuzzed projects
```

Comment:
after this pr is merged, @ethanbwang please use the full list of projects to get a summary of all our supported projects in C.
The command would be

```sh
python3 scripts/analyze.py
```